### ISSUE TITLE
Error Downloading Images

### ISSUE THREAD

**Issue by ayaybra** [garrynewman/SteamScreenshotDownloader#2]
The program is able to detect how many screenshots I have but I don't think it's actually downloading the screenshots. Every call to download a specific screenshot has a response of "The given header was not found."

**Comment by hifocus** [comment]
> The program is able to detect how many screenshots I have but I don't think it's actually downloading the screenshots. Every call to download a specific screenshot has a response of "The given header was not found."

Yes I have this issue as well.

![image](https://user-images.githubusercontent.com/30528433/143918375-072219b2-56a9-4ea0-9a46-4706ab66d844.png)

**Comment by RavusSapiens** [comment]
The Problem still persists. It seems to detect all 900+ images, but only downloads one.
![image](https://user-images.githubusercontent.com/116229214/196818612-5d7ec415-ba07-41e5-b87c-4fe40fcb91e8.png)

**Comment by afterdelight** [comment]
https://github.com/garrynewman/SteamScreenshotDownloader/pull/3

**Comment by Lightning2X** [comment]
> #3

Yep it should fix it :) Im gonna make a version where it somehow downloads the caption too hopefully. I'll see if graphics in C# is doable

**Comment by afterdelight** [comment]
nice, could you do an update to download steam artworks too like `https://steamcommunity.com/id/sonicswed95/images/`

**Comment by Lightning2X** [comment]
Sure should be doable, Can you make an issue at https://github.com/Lightning2X/SteamScreenshotDownloader so I can keep track of it?

**Comment by afterdelight** [comment]
Sure, but first you need to change your fork setting so i can open an issue at https://github.com/Lightning2X/SteamScreenshotDownloader

**Comment by Lightning2X** [comment]
> Sure, but first you need to change your fork setting so i can open an issue at https://github.com/Lightning2X/SteamScreenshotDownloader

My bad! Enabled it now

**Comment by afterdelight** [comment]
I've opened the issue at https://github.com/Lightning2X/SteamScreenshotDownloader/issues/1
